src/features/projects/components/projectDetails/TopProjectReports.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styles from './../../styles/ProjectDetails.module.scss';
+import VerifiedIcon from '@mui/icons-material/Verified';
+import { getPDFFile } from '../../../../utils/getImageURL';
+import parse from 'date-fns/parse';
+import format from 'date-fns/format';
+import { Trans } from 'react-i18next';
+import { useTranslation } from 'next-i18next';
+
+interface Props {
+  data: object[];
+}
+export default function TopProjectReports(data: Props) {
+  const { t, ready } = useTranslation(['common']);
+  return (
+    ready && (
+      <>
+        <div className={styles.reports_container}>
+          <VerifiedIcon sx={{ color: '#42A5F5' }} />
+          <div className={styles.reports_description}>
+            {data?.data?.map((review) => (
+              <div key={review.id}>
+                <p id="child-modal-description">
+                  <Trans i18nKey="common:reviewInfo">
```

Comment:
For some reason I see the text twice for https://planet-webapp-4kuqh5hnd-planetapp.vercel.app/dingopocket?site=site_QFqrJongyVyr9Z6

<img width="436" alt="Bildschirm­foto 2023-03-25 um 11 29 46" src="https://user-images.githubusercontent.com/1532418/227712313-812568a3-9c15-4ddf-9edc-11957bfb08b5.png">

<img width="659" alt="Bildschirm­foto 2023-03-25 um 11 27 01" src="https://user-images.githubusercontent.com/1532418/227712317-d19b0050-6645-4ee8-a87d-1df25d39a38a.png">

Reply:
Sorry, this is because there are really two reviews on this project.